docs/source/file_ops.rst

```diff
@@ -0,0 +1,11 @@
+
+file_ops
+========
+**file_ops** contains a variety of file related operations commonly used by the DAP team
+
+.. autofunction:: file_ops._convert_str_to_pathlib_path
```

Comment:
I'm not 100% sure how other packages handle this, but I think you could either not include internal functions or put them at the end.